cargo-dist/templates/installer/installer.ps1.j2

```diff
@@ -47,6 +50,47 @@ param (
 $app_name = '{{ app_name }}'
 $app_version = '{{ app_version }}'
 
+$install_path = $null
+# If you're reading this install script generated by an app, and
+# this looks strange to you, it's because it's possible for there
+# to be multiple generated paths here. If you're only seeing one
+# $install_path assignment, it's because the app you're looking at
+# has been configured with a single install path.
+{% for install_path in install_paths %}
+if (-Not $install_path) {
```

Comment:
...we should really look into if we're supposed to be wrapping these in quotes like we do in shell...

Reply:
I was wondering the same thing. It's a bit inconsistent about that atm